search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java

```diff
@@ -160,8 +161,10 @@ public enum ResourceParameter implements ParameterKey<ResourceParameter> {
     ID(KEYWORD, ANY_OF, IDENTIFIER_KEYWORD),
     ID_NOT(KEYWORD, NOT_ANY_OF, IDENTIFIER_KEYWORD),
     ID_SHOULD(TEXT, ANY_OF, IDENTIFIER_KEYWORD),
-    INSTANCE_TYPE(KEYWORD, ANY_OF, PUBLICATION_INSTANCE_TYPE, PATTERN_IS_CATEGORY_KEYS, null, null),
-    INSTANCE_TYPE_NOT(KEYWORD, NOT_ANY_OF, PUBLICATION_INSTANCE_TYPE, PATTERN_IS_CATEGORY_NOT_KEYS, null, null),
+    INSTANCE_TYPE(KEYWORD, ANY_OF, PUBLICATION_INSTANCE_TYPE, PATTERN_IS_CATEGORY_KEYS, null, null,null),
+    INSTANCE_TYPE_NOT(KEYWORD, NOT_ANY_OF, PUBLICATION_INSTANCE_TYPE, PATTERN_IS_CATEGORY_NOT_KEYS, null, null,null),
+    INSTANCE_TYPE_PART_OF(PART_OF, ALL_OF, INSTANCE_TYPE),
```

Comment:
not ANY_OF?

Reply:
For partOf & hasParts queries, it is the subquery that has either any_of or all_of,....

I will make update to make it clearer...